unattended-upgrade

```diff
@@ -1640,6 +1646,26 @@ def run(options,             # type: Options
     if should_stop():
         return UnattendedUpgradesResult(False)
 
+    # check to see if want to auto-upgrade the devel release
+    if apt_pkg.config.find("Unattended-Upgrade::DevRelease") == "auto":
+        try:
+            import distro_info
+            devel = (distro_info.DistroInfo(DISTRO_ID) .
+                     devel(result="object"))
+        except Exception:
+            logging.warning("Could not figure out development release")
+        else:
+            if ((devel.series == DISTRO_CODENAME and
+                 devel.release is not None and
+                 devel.release - date.today() >= DEVEL_UNTIL_RELEASE)):
+                logging.info(_("Not running on this development "
+                               "release before %s") %
```

Comment:
@vorlonofportland U-u in did that for years thus reporting via logging.info() is not a regression thus it is not a blocker for merging for me.

Reply:
Probably it would be useful to pick tests from https://github.com/mvo5/unattended-upgrades/pull/54 (latest delta can be genarated by diffing to the ubuntu/bionic branch) and adding a test using faketime to not change test behavior based on time.